### DISCUSSION TITLE
Adding support for Autobahn API related to German motorways.

### DISCUSSION THREAD

**Discussion by Selaron** [osmandapp/OsmAnd#12460] [Ideas]
What would be the best way to integrate the Autobahn API into osmand providing details on German motorway state like
- road works
- traffic
- web cams
- recharging spots
- resting spots
- blocked roads
?

Would be quite hot to have osmand calculate fastest route considering traffic jams while keeping your electric car in range of recharging spots.

**Comment by openfnord** [comment]
You like to add number plate readers,
so you could add them to avoid POI
if you want to travel in privacy
